### PR TITLE
intl: EN: Remove plural s for "information"

### PR DESCRIPTION
**Pull request by olof-nord**
Hello Devolo team!

Usually the usage of the English word "information" is uncountable.

Me reading this message made me assume it was a translation mistake, the Wiktionary [1] entry lists it as _usually_ uncountable. 

With this PR I can only talk for myself, and perhaps if you have a translation team ask them what they say?

btw: many thanks for the recent updates, now I can run the application locally without any problems.

[1] https://en.wiktionary.org/wiki/information

### CODE REVIEW

File: lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -77,7 +77,7 @@ class MessageLookup extends MessageLookupByLibrary {
         "chooseTheAppTheme":
             MessageLookupByLibrary.simpleMessage("Choose the app theme"),
         "cockpitSupportInformationBody": MessageLookupByLibrary.simpleMessage(
-            "The support informations have been created and are now available to you. Please select an action."),
+            "The support information have been created and are now available to you. Please select an action."),
```
Comment (Shutgun) [inline lib/generated/intl/messages_en.dart]:
```suggestion
            "The support information has been created and is now available to you. Please select an action."),
```

File: lib/l10n/intl_en.arb [hunk 2]
```diff
@@ -99,9 +99,9 @@
   "pleaseEnterYourMailAddress": "Enter your email address",
   "contactInfo": "Contact information",
   "yourEmailAddress": "Your email address",
-  "LoadCockpitSupportInformationBody": "Support informations are getting generated",
-  "SendCockpitSupportInformationBody": "Support informations are sent to devolo",
-  "cockpitSupportInformationBody": "The support informations have been created and are now available to you. Please select an action.",
+  "LoadCockpitSupportInformationBody": "Support information is getting generated",
+  "SendCockpitSupportInformationBody": "Support information is sent to devolo",
+  "cockpitSupportInformationBody": "The support information have been created and are now available to you. Please select an action.",
```
Comment (Shutgun) [inline lib/l10n/intl_en.arb]:
```suggestion
  "cockpitSupportInformationBody": "The support information has been created and is now available to you. Please select an action.",
```

**Review by Shutgun — CHANGES_REQUESTED** [review]
Nice catch, just two small suggestions.

**Comment by olof-nord** [comment]
Happy to hear positive feedback.
Suggestions added, thanks for the review.

**Review by Shutgun — APPROVED** [review]

**Comment by olof-nord** [comment]
A quick note regarding merging: "Only those with write access to this repository can merge pull requests.".
If and when this should be merged, the assistance of someone with write access is required.

**Comment by Shutgun** [comment]
Yeah, it will be merged soon, I just want @Calorine or @julienAhn to have a chance to review it, too.

**Review by julienAhn — APPROVED** [review]
This looks good. Thank you for contributing!

**Pull request merged**